src/app/zap-templates/zcl/data-model/chip/joint-fabric-administrator-cluster.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ICACResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="InvalidPublicKey" value="0x01"/>
+    <item name="InvalidICAC" value="0x02"/>
+  </enum>
+
+  <cluster>
```

Comment:
Where is TransferAnchorResponseStatusEnum?  I see it in the spec PR this claims to implement, but not here.

Reply:
Anchor transfer feature remains provisional in 1.5 and it's a candidate for 1.6. This PR does not implement the anchor transfer feature, which will be handled in a future PR.